features/support/formatter.rb

```diff
@@ -34,7 +34,7 @@ def initialize(runtime, path_or_io, options)
 
       #
 
-      def before_features(features)
+      def before_features(_features)
```

Comment:
I take that back, it looks like we aren't referencing their implementation as a sub-class.  Carry on.